src/Database/Schema/SalesforceSchema.php

```diff
@@ -45,7 +47,7 @@ public function listTables()
     }
 
     /**
-     * Custom function that queries the datasource for the table list as SOQL doesn't accept
+     * Custom function that queries the datasource for the table list as SQL doesn't accept
```

Comment:
This was actually SOQL on purpose; that's Salesforce's custom not-quite-standard SQL variant.